### PR TITLE
Create schandra.md

### PR DESCRIPTION
**Pull request by sreechandran**
Adding my favourite colour and food

### CODE REVIEW

**Comment by gabarmas-ibm** [comment]
You are missing the food though

File: bios/schandra.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Creating a file to put in my favourites
+Fav Colour : red
```
Comment (gabarmas-ibm) [inline bios/schandra.md]: Red is a good choice

**Comment by sreechandran** [comment]
@gabarmas My favourite cusine is Ethiopian.

File: bios/schandra.md (hunk 1 above)
Comment (sreechandran) [inline bios/schandra.md]: Thank you

**Comment by gabarmas-ibm** [comment]
@sreechandran Best choice. Sushi is too mainstream

**Comment by sreechandran** [comment]
@karenbruzzo Hello !

File: bios/schandra.md (hunk 1 above)
Comment (gabarmas-ibm) [inline bios/schandra.md]: You are welcome

**Comment by sreechandran** [comment]
closing #119 pull request

**Pull request merged**